website/_docs/watchman-make.md

```diff
@@ -89,3 +89,19 @@ $ watchman-make -p '*.c' '*.h' 'Makefile*' 'tests/**/*.py' 'tests/**/*.c' -t all
 this will execute `make all integration` if you change any top level `*.c` file
 *or* test source file.
 
+### Run Scripts
+
```

Comment:
Insert:

```
*Since 4.8.*
```

here so that folks know which version this will appear in.  We're prepping to tag this right now; just have a couple of things to tidy up, so it should be out by the end of next week, assuming things go to plan.